kolibri/plugins/facility/assets/src/views/users/UsersRootPage/index.vue

```diff
@@ -307,6 +313,10 @@
     },
     methods: {
       overrideRoute,
+      navigateToSidePanel(sidePanelName) {
```

Comment:
Could you define this in the `setup()` function so that the implementations are similar between this and the new users page? The root users  & new users pages are so similar that I think most changes to one will result in a change in the other, so the more consistently they're structured the less friction for making those changes down the road.